### PR TITLE
Expose vehicle list request and allow choice of vehicle to park

### PR DESCRIPTION
**Pull request by ebk46**
feat(service): expose request to fetch vehicle list
feat(service): enable choice of vehicle to park

### CODE REVIEW

File: parknyc/service.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 from base64 import b64encode
 from datetime import datetime, timedelta
 import uuid
+import time
 
-from parknyc.exceptions import *  # noqa
+from nycpark.exceptions import *  # noqa
```
Comment (alecxe) [inline parknyc/service.py]: @ebk46 this should be `parknyc` as it is the package name here?

**Review by alecxe — COMMENTED** [review]
Appreciate the pull request, Evan!

File: parknyc/service.py (hunk 1 above)
Comment (ebk46) [reply parknyc/service.py]: Wow that's embarrassing, yes good catch, thanks.

Comment (ebk46) [reply parknyc/service.py]: Made the change.

**Pull request merged**